system/config/Config.cfc

```diff
@@ -1242,6 +1244,29 @@ component {
 		}
 	}
 
+	private void function __setupLocaleSettings() {
+		settings.datetime.regionDefaults = {
+			  uk = {
+				  short_date_format = "dd/mm/yyyy"
+				, long_date_format = "DMY"
+			  }
+			, eu = {
+				  short_date_format = "dd.mm.yyyy"
+				, long_date_format = "DMY"
+			  }
+			, us = {
+				  short_date_format = "mm/dd/yyyy"
+				, long_date_format = "MDY"
+			  }
```

Comment:
Should our region defaults specify 12/24 hour? e.g. 24h for eu, 12h for uk and us. If so, will need to implement elsewhere when we populate these defaults